File: Modul 4/Soal 5/PRAK405-2410817220018-ANGGRAENI DWI ZAHRA.py
n = int(input("Masukkan Jumlah Baris: "))
kelipatan = int(input("Masukkan Kelipatan: "))

total = 0  

for i in range(1, n + 1):
    baris_total = 0  
    ekspresi = ""  

    for j in range(i, 0, -1):
        baris_total += j * kelipatan
        if j != i:
            ekspresi += " + "
        ekspresi += f"({j} * {kelipatan})"
    
    print(ekspresi, "=", baris_total)
    
    total += baris_total
print(total)

File: Modul 4/Soal 5/PRAK405-2410817220018-ANGGRAENI-DWI-ZAHRA.c
#include <stdio.h>

int main() {
    int n, kelipatan;
    int total = 0;  

    printf("Masukkan jumlah baris: ");
    scanf("%d", &n);
    printf("Masukkan kelipatan: ");
    scanf("%d", &kelipatan);

    for (int i = 1; i <= n; i++) {
        int baris_total = 0;  

        for (int j = i; j >= 1; j--) {
            baris_total += j * kelipatan;
            
            if (j != i) {
                printf(" + ");
            }
            printf("(%d * %d)", j, kelipatan);
        }

        printf(" = %d\n", baris_total);
        
        total += baris_total;
    }

    printf("%d\n", total);

    return 0;
}